minify/minify_test.go

```diff
@@ -0,0 +1,594 @@
+package minify
```

Comment:
The github actions CI file for running these tests is missing.

Reply:
Added to  .github/dependabot.yml and  .github/workflows/test.yml